lib/vmdb/inflections.rb

```diff
@@ -46,7 +46,24 @@ def self.load_core_inflections
         inflect.plural(/quota$/, "quotas")
         inflect.irregular("container_quota", "container_quotas")
 
+        inflect.acronym('DBus')
+        inflect.acronym('DDF')
+        inflect.acronym('GHT')
+        inflect.acronym('IPMI')
         inflect.acronym('ManageIQ')
+        inflect.acronym('MKS')
+        inflect.acronym('OAuth')
+        inflect.acronym('REXML')
+        inflect.acronym('SAP')
+        inflect.acronym('SSL')
+        inflect.acronym('UI')
+      end
+
+      # ActiveSupport::Inflector.inflections doesn't allow us to blanket tell how to autoload a constant from a file
+      # This works around the UI acronym above.
+      Rails.autoloaders.each do |autoloader|
+        autoloader.inflector.inflect('miq_ui_worker' => "MiqUiWorker")
```

Comment:
```
ManageIQ::Providers::Ovirt::InfraManager::AdminUI
ManageIQ::UI::Classic
```

uses the `UI` acronym.  The `MiqUiWorker` could be renamed and a migration be written to rename the old name to the new `MiqUIWorker`.  Then, all places we have would need to be renamed to reference the new location.

Reply:
Discussed with @kbrock, we should extract this whole file's changes on master as they don't depend on zeitwerk so we can merge them now.

Reply:
Extracted to https://github.com/ManageIQ/manageiq/pull/22620